packages/visual-editor/src/utils/schema/getSchema.ts

```diff
@@ -22,6 +22,11 @@ export const getSchema = (data: TemplateRenderProps): Record<string, any> => {
   // Move path to the document for schema resolution
   document.path = data.path;
 
+  // Set placeholder site domain for non-production domains
+  if (!document.siteDomain) {
+    document.siteDomain = "<siteDomain>";
```

Comment:
I guess this is related to my previous question and the comment here sort of answers it. Do we want this placeholder string?